src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
 public @interface Component {
```

Comment:
Don't forget to add @ Target to custom annotations.

In the video about annotations it is not necessary, but in common mistakes it seems to be necessary))